<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLWindowLevelWidget</class>
 <widget class="QWidget" name="qMRMLWindowLevelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>61</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Window Level</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Window/Level:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="AutoManualComboBox">
     <item>
      <property name="text">
       <string>Auto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manual Min/Max</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ctkDoubleSpinBox" name="WindowSpinBox">
       <property name="prefix">
        <string>W: </string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="decimalsOption">
        <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSpinBox" name="MinSpinBox">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="decimalsOption">
        <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleRangeSlider" name="WindowLevelRangeSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSpinBox" name="MaxSpinBox">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="decimalsOption">
        <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSpinBox" name="LevelSpinBox">
       <property name="prefix">
        <string>L: </string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="decimalsOption">
        <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="RangeButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:Icons/SliceMoreOptions.png</normaloff>:Icons/SliceMoreOptions.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDoubleRangeSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleRangeSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
